src/Cli.Tests/InitTests.cs

```diff
@@ -409,6 +409,73 @@ public Task GraphQLPathWithoutStartingSlashWillHaveItAdded()
             return ExecuteVerifyTest(options);
         }
 
+        /// <summary>
+        /// Test to validate the config is correctly generated with different database types and various options for --graphql.nested-create.enabled flag.
```

Comment:
Can you elaborate on what you're verifying? contents of generated config? are you also validating behavior when dab get various db types other than mssql?

Reply:
I'm validating the contents of the generated config file. I've updated the description